app/controllers/fuel_controller.rb

```diff
@@ -14,4 +14,44 @@ def s3_bucket
     )
   end
 
+  def self.paginated_action(options={})
+    before_filter(options) do |controller|
+
+      # Set defaults
+      @pagination_current_page = params[:page] ? params[:page] : 1
+      @pagination_per_page = nil
```

Comment:
Perhaps I'm reading this wrong, but shouldn't this `pagination_per_page` var be set to `config.paginates_per` when not using the `header_pagination`-style pagination?